src/frontend/src/tests/instance_test.rs

```diff
@@ -913,27 +913,27 @@ async fn test_information_schema(instance: Arc<dyn MockInstance>) {
 
     // User can only see information schema under current catalog.
     // A necessary requirement to GreptimeCloud.
-    let sql = "select table_catalog, table_schema, table_name, table_type from information_schema.tables where table_type != 'SYSTEM VIEW' order by table_name";
+    let sql = "select table_catalog, table_schema, table_name, table_type, table_id, engine from information_schema.tables where table_type != 'SYSTEM VIEW' order by table_name";
 
     let output = execute_sql(&instance, sql).await;
     let expected = "\
-+---------------+--------------------+------------+------------+
-| table_catalog | table_schema       | table_name | table_type |
-+---------------+--------------------+------------+------------+
-| greptime      | public             | numbers    | BASE TABLE |
-| greptime      | public             | scripts    | BASE TABLE |
-| greptime      | information_schema | tables     | VIEW       |
-+---------------+--------------------+------------+------------+";
++---------------+--------------------+------------+------------+----------+-------------+
+| table_catalog | table_schema       | table_name | table_type | table_id | engine      |
++---------------+--------------------+------------+------------+----------+-------------+
+| greptime      | public             | numbers    | BASE TABLE | 1        | test_engine |
+| greptime      | public             | scripts    | BASE TABLE | 1        | mito        |
```

Comment:
In local machine the scripts's `table_id` is 1, but in github test the `table_id` is 1024.

Reply:
I checked the CI log. Did you check it passed the `instance_test::test_information_schema::case_2_test_with_distributed` test? (If you are using the vs code, you will find multiple `run test` text buttons) . 

```
  PASS [   0.074s] frontend tests::instance_test::test_information_schema::case_1_test_with_standalone
   FAIL [   1.437s] frontend tests::instance_test::test_information_schema::case_2_test_with_distributed
```

I guess it's due to the inconsistent catalog behaviors between the distributed and standalone modes.

Reply:
Looks like this test runs the same cases both for standalone and distributed mode. What about ignoring the table id column in the tests? The actual value of the table id in the test is not so significant now.

Reply:
You can add a method to `MockInstance` to return whether it is distributed like this: https://github.com/MichaelScofield/greptimedb/blob/1380772cc38c6dea4b6e5ae132680fecc7b97b41/src/frontend/src/tests/test_util.rs#L30

And set different expected values for each mode: https://github.com/MichaelScofield/greptimedb/blob/1380772cc38c6dea4b6e5ae132680fecc7b97b41/src/frontend/src/tests/instance_test.rs#L328